config/initializers/omniauth.rb

```diff
@@ -1,10 +1,11 @@
 Rails.application.config.middleware.use OmniAuth::Builder do
+  protocol = ENV['RAILS_ENV'] == 'development' ? 'http://' : 'https://'
```

Comment:
You could simplify this with:
```
protocol = Rails.env.development? ? 'http://' : 'https://'
```

It might also be clearer if the `protocol` was simply `http` or `https` rather than the protocol followed by the colon & trailing slashes. `http` is a `protocol`, while `http://` is a protocol followed by some punctuation. 

As those are just stylistic niggles, I'll approve and leave to your discretion.

Reply:
its more dry to include the colon and trailing slashes thats why I did it.
But I'll add this protocol = Rails.env.development? ? 'http://' : 'https://'